server.js

```diff
@@ -10,6 +10,19 @@ app.set('view engine', 'hbs');
 app.use(express.static(path.join(__dirname, 'public')));
 
 // add your routes here :)
+app.get('/:code', function (req,res) {
+	const url = 'http://localhost:8080/code/' + req.params.code;
+	const url2 = 'http://numbersapi.com/' + req.params.code;
+	const promise1 = fetch(url).then((res) => res.json());
+	const promise2 = fetch(url2).then((res) => res.text());
```

Comment:
I'm not a fan of making variables so generic, when you come across them later after their initialization you might have forgotten their context. Something like `localhostUrl` and `numbersApiUrl` (and then maybe `localhostUrlPromise`) would make them easy to keep track of. It's no big deal in production as you should minify/uglify everything so the bytes down the line won't be a factor :)